notes/ccc-project-notes-1.html

```diff
@@ -0,0 +1,92 @@
+"represents an independent line of development | serves as an abstraction for the edit/stage/commit process in Git"
+</p>
+<strong>branch naming</strong> convention
+<li>chore - changing simple things like a name</li>
+<li>feature - new compononet</li>
+<li>bugfix- broken code that needs to be fixed</li>
+<p><strong>hotfix / quick-fix</strong><br>
+fixes up a production bug fast and is applied
+</p>
+<hr>
+<h2>COMMANDS</h2>
+<p><strong>to create a new branch</strong><br>
+<code>git checkout -b feature/blog</code>
+</p>
+<p><strong>to go to branch</strong><br>
+<code>git checkout feature/blog</code>
+</p>
+<p><strong>to delete branch</strong><br>
+<code>git branch -d feature/blog</code>
+</p>
+<p><strong>to see all branches</strong><br>
+<code>git branch</code>
+</p>
+<p><strong><code>*</code></strong><br>
+denotes current branch
+</p>
+<blockquote>"feature/chat" - just a naming convention - just a string - not a containing folder</blockquote>
+<blockquote>be careful making new branches because it's based on where you are right now - the current branch
+</blockquote>
+<blockquote>always go back to master and make the new thing - should always be based on the master - should always create a branch off of the master<br></blockquote>
+<strong>PULL REQUEST</strong><br>
+<code>git pull</code><br>
+commonly referred to as "PR"<br>
+is to grab all the updated version from the master on github
+making local copy up to date with one published on github<br>
+<hr>
+<p><strong>MERGE vs. REBASE</strong></p>
+<ul style="list-style-type: none;">
+	<li>NEW COMPONENTS</li>
+	<li>rebase maintains history</li>
+	<li>merge messes up history</li>
+</ul>
+<hr>
+<p><strong>CONFLICTS</strong></p>
+<p style="text-indent: 25px;"><code>git stash save</code></p>
+<ul style="list-style-type: none;">
+	<li>temporary stash</li>
+	<li>do not restart computer or it will delete it</li>
+</ul>
+<p style="text-indent: 25px;"><code>git stash pop</code></p>
+<ul style="list-style-type: none;">
+	<li>bring it back out to work on</li>
+	<li>similar to rebase and merging</li>
+	<li>is used for working with stuff locally</li>
+</ul>
+<hr>
+<strong>CHAT FEATURE</strong><br>
+<p style="text-indent: 25px;"><strong>Next.js</strong></p>
+<ul style="list-style-type: none;">
+	<li>react with extra features</li>
+	<li>instead of HTML, CSS, JS as homepage > gives 'pages'</li>
+	<li>folder with each file containing view</li>
+	<li>removes boilerplate code</li>
+	<li>tech that big companies are switching over to</li>
+	<li>better performance and cost & time efficient</li>
+</ul>
+<p style="text-indent: 25px;"><strong>GraphQL</strong></p>
+<ul style="list-style-type: none;">
+	<li>developed by facebook</li>
+	<li>only gives the data back that you need</li>
+</ul>
+<p style="text-indent: 25px;"><strong>Hasura</strong></p>
+<ul style="list-style-type: none;">
+	<li>site/UI to build stuff really quickly</li>
+	<li>will handle relationships really well</li>
+</ul>
+

```

Comment:
This is usually found in a README, we don't want dev related instructions in an HTML file, plus it's harder to find.

I would suggest using GitHub README formatting syntax to still have a similar effect.
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

You could add it to `notes/README` for now